examples/react-app/webpack.config.js

```diff
@@ -19,6 +19,10 @@ const config = {
           },
         ],
       },
+      {
+        test: /\.css$/i,
+        use: ['style-loader', 'css-loader'],
+      },
```

Comment:
The Webpack config needed special permissions to import the polyfill CSS correctly. I'm not a Webpack Wizard, so there's probably a better way to handle this, but since I was just proving an IE solution could be reached, I went with what worked.